src/pytest_ansible/host_manager/__init__.py

```diff
@@ -40,14 +40,14 @@ def get_extra_inventory_hosts(self, host_pattern=None):
                         host_pattern
                     )
                 ]
-        except:
```

Comment:
Make sure these return ValueErrors, not any other errors. Else it won't be caught.